SamplesDashboard/SamplesDashboard/ClientApp/src/components/DetailList.tsx

```diff
@@ -0,0 +1,186 @@
+﻿import * as React from 'react';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { DetailsList, DetailsListLayoutMode, Selection, IColumn, SelectionMode } from 'office-ui-fabric-react/lib/DetailsList';
+import { MarqueeSelection } from 'office-ui-fabric-react/lib/MarqueeSelection';
+import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
+import { Sticky, StickyPositionType } from 'office-ui-fabric-react/lib/Sticky';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
+import { mergeStyles, mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+import { Announced } from 'office-ui-fabric-react/lib/Announced';
+
+const detailListClass = mergeStyles({
+    display: 'block',
+    marginBottom: '10px'
+
+});
+const classNames = mergeStyleSets({
+    wrapper: {
+        height: '80vh',
+        position: 'relative',
+        display: 'flex',
+        flexWrap: 'wrap'
+    }
+});
+
+export interface IListDataItem {
+    key: number;
+    name: string;
+    owner: string;
+    status: string;
+    language: string;
+    pullRequests: number;
+    issues: number;
+    stars: number;
+    featureArea: string;
+    securityAlerts: string;
+}
+
+export interface IListDataState {
+    columns: IColumn[];
+    items: IListDataItem[];
+    selectionDetails: string;
+    announcedMessage?: string;
+}
+
+export default class ListData extends React.Component<{}, IListDataState> {
+    private _selection: Selection;
+    private _allItems: IListDataItem[];
+
+    constructor(props: {}) {
+        super(props);
+
+        this._selection = new Selection({
+        onSelectionChanged: () => this.setState({ selectionDetails: this._getSelectionDetails() })
+        });
+        // Populate with items for demos.
+        this._allItems = [];
+        const columns: IColumn[] = [
+            { key: 'column1', name: 'Name', fieldName: 'name', minWidth: 200, maxWidth: 300, isRowHeader: true, isResizable: true, isSorted: true, isSortedDescending: false, onColumnClick: this._onColumnClick },
```

Comment:
Rename the columns to something more readable.. You can repeat the fieldName value in this case

Reply:
Do you mean the column key or the column name??